### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by thedavidprice**
Related to #1953

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 0.x.x   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+If you discover a potential security issue, do let us know as soon as possible. We'll work toward resolution as quickly as possible, so please provide us with a reasonable amount of time before disclosure to the public or a third-party.
```
Comment (Tobbe) [inline SECURITY.md:12]:
```suggestion
If you discover a potential security issue, do let us know as soon as possible. We'll work toward a resolution as quickly as possible, so please provide us with a reasonable amount of time before disclosure to the public or a third-party.
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Contact us at **EMAIL HERE**
```
Comment (Tobbe) [inline SECURITY.md:14]: Don't forget to insert actual email

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Security Policy
+The Redwood repo is scanned frequently for code and dependency vulnerabilities. Notifications are received by the Redwood Core Team members, who assess risk, prioritize, and determine a remediation plan. Typically this process involves a package update or patch release and is resolved within 24 hours of notification.
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 0.x.x   | :white_check_mark: |
```
Comment (Tobbe) [inline SECURITY.md:8]: I'm not sure I understand this table. What is it for? Is this how it should look?

**Review by Tobbe — CHANGES_REQUESTED** [review]

**Comment by thedavidprice** [comment]
- [ ] add redwoodjs.com/security.txt 
    - add link to Security.md
    - add email to contact us
 - [ ] add section somewhere that shows up in Search

**Review by cannikin — APPROVED** [review]
Other than the missing version and **EMAIL HERE** looks good to me!

File: SECURITY.md (hunk 2 above)
Comment (thedavidprice) [reply SECURITY.md:14]:
```suggestion
Contact us at [security@redwoodjs.com](mailto:security@redwoodjs.com)
```

**Pull request merged**